### ISSUE TITLE
accessibility of CSS styles (eg, font size and color in header bar)

### ISSUE THREAD

**Issue by RichMorin** [diaspora/diaspora#7974]
The header bar uses a small, medium-gray font on a black background. This may be difficult for folks with low vision to read. More generally, has the UI been run through an accessibility audit (eg, the one in Chrome)?

-r

**Comment by Flaburgan** [comment]
Hi and thank you for taking time to open an issue! Diaspora does have some accessibility issue. It didn't receive an accessibility audit at the moment. Unfortunately, no-one stepped up to do that job, and as only volunteers are working on the project, we can't guarantee any date for this problem to be tackled.
If you want to help, you're welcome!
